src/i18n.js

```diff
@@ -0,0 +1,27 @@
+import Vue from "vue";
```

Comment:
Check this https://github.com/linagora/openpaas-dashboard/blob/master/src/i18n/index.js

This is simpler and lazy load files also.